avm/res/document-db/database-account/CHANGELOG.md

```diff
@@ -2,6 +2,22 @@
 
 The latest version of the changelog can be found [here](https://github.com/Azure/bicep-registry-modules/blob/main/avm/res/document-db/database-account/CHANGELOG.md).
 
+## 0.18.0
+
+### Changes
+
+- **Added support for Cassandra API resources** - First version to include Cassandra keyspaces, tables, views, role assignments, and role definitions for Azure Cosmos DB
+- Import `tableType` and `viewType` from `cassandra-keyspace` module to eliminate code duplication
+- Reduced template size by approximately 85 lines through type import architecture
+- Established single source of truth for Cassandra resource types across modules
```

Comment:
The changelog should focus on changes that were introduced on top of the last published version (i.e., what's in main) and affects consumers of the module.
This is correct for the first item, but the remaining 3 don't provide the same value unfrotuantely. 

Top down, I'd recommend to mention
- That you added & export new types in the top-level main.bicep (and which, like `cassandraStandaloneRoleAssignmentType`)
- That you added support for cassandra child resource types (and which, like `cassandra-role-definition`)
- The renaming of pre-existing parameters as breaking changes (as you did below)